mlir/include/mlir/Target/LLVMIR/LLVMImportInterface.h

```diff
@@ -98,6 +98,12 @@ class LLVMImportDialectInterface
 /// efficient dispatch.
 class LLVMImportInterface
     : public DialectInterfaceCollection<LLVMImportDialectInterface> {
+private:
+  // Generate llvm.call_intrinsic when no supporting dialect available.
+  LogicalResult
+  convertUnregisteredIntrinsic(OpBuilder &builder, llvm::CallInst *inst,
```

Comment:
Can you move the function at the end of the class declaration in the private section with the member variables. Additionally it may make sense to make it a static function since it doesn't seem to access any members.